services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -156,7 +158,11 @@ export class NoticeOfWorkApplication extends Component {
   };
 
   toggleEditMode = () => {
-    this.setState((prevState) => ({ isViewMode: !prevState.isViewMode }));
+    this.setState((prevState) => ({
+      isViewMode: !prevState.isViewMode,
+      isEditing: !prevState.isEditing,
+      menuVisible: false,
+    }));
```

Comment:
What is the difference between `isViewMode` and `isEditing` ? They are the same but opposite, can we use one?

Reply:
Changed to just use isViewMode.